labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -33,8 +34,17 @@ object CollectionExercise01 {
    *
    */
   def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+    val input = "ejp mysljylc kd kxveddknmc re jsicpdrysi rbcpc ypc rtcsra dkh wyfrepkym veddknkmkrkcd de kr kd eoya kw aej tysr re ujdr lkgc jv"
+    val output = "our language is impossible to understand there are twenty six factorial possibilities so it is okay if you want to just give up"
+    var letterMap = stringsToMap(input, output)    
+    
+    lines.map(str => str.map(c => letterMap(c)))
```

Comment:
Not all the letters are represented in the input sequence. This function won’t correctly decode `"qee"` into `"zoo"`. I’m not sure if anybody actually handled that problem well, though: I hardcoded the whole map; Prof. Ben hardcoded a few codes which were not all given in the hints at the start of the function. The sample solution doesn’t seem to do either, but I didn’t quite understand what it did.